Documentation/reportgenerationqueries.md

```diff
@@ -11,7 +11,8 @@ When writing a [report](report.md) you can specify the query it will use by sett
 - `columns`: A list of columns that match the schema of the results of the query. The order of these columns must match the order of the columns returned by the SELECT statement. Columns have 3 fields, `name`, `type`, and `unit`. Each field is covered in more detail below.
   - `name`: This is the name of the column returned in the `SELECT` statement.
   - `type`: This is the [Hive][hive-types] column type. Currently due to implementation details, column types are expressed using hive types. In the future, this will likely be switched to using the Presto native types. This also has an effect that queries with columns containing complex types such as `maps` or `arrays` cannot be used by `Reports` or `ScheduledReports`.
-  - `unit`:
+  - `unit`: Unit refers to the measurement of the column.
+  - `tableHidden`: Takes a Bool the either shows or hides the column. This determines whether the column will be displayed or not in reports.
```

Comment:
This probably should be more details about what it hides/shows and when.

`Takes a boolean, when true, hides the column from report results depending on the format and endpoint. If true, the /api/v2/reports/$reportName/table endpoint will filter tableHidden columns out of the results. Columns with TableHidden set to true are always hidden for tabular and CSV outputs.`

Reply:
Actually link to the API docs on the v2 table endpoint for the second sentence instead. 

Just say `See [api docs for details][apiTable].` and set `[apiTable]: api.md#v2-reports-table` at the bottom of the file.

So the full line would be:
```
Takes a boolean, when true, hides the column from report results depending on the format and endpoint. See [api docs for details][apiTable].
```

With this at the bottom of the file:
```
[apiTable]: api.md#v2-reports-table
```